crates/aptos/src/common/types.rs

```diff
@@ -1090,6 +1125,14 @@ pub struct MovePackageDir {
     /// Select bytecode, language, compiler for Move 2
     #[clap(long)]
     pub move_2: bool,
+
+    /// Select optimization level
```

Comment:
Would be good to say what the different options are, and briefly say which option is what. Because this is what shows up on doing `aptos move compile --help`.

Reply:
Done.

Reply:
If I run: `aptos move compile --optimize=none`, I do not get any error, even though compiler v1 is run (by default), and the option does not make sense for that compiler.

However, if I run `aptos move compile --compiler-version=1 --optimize=none`, then I do get an error saying this flag is not compatible with compiler v1, which is the expected behavior.

So, there may be some plumbing issue here.

Reply:
Fixed.